socorro/unittest/signature/test_rules.py

```diff
@@ -1575,6 +1577,33 @@ def test_action_success(self):
         expected = ['Signature replaced with an IPC Channel Error, was: "foo::bar"']
         assert notes == expected
 
+    def test_action_non_ascii(self):
+        rule = SignatureIPCChannelError()
+
+        raw_crash = {
+            'ipc_channel_error': u'あわわ'
```

Comment:
Instead of including unicode characters directly, I'd use the `\x` or `\u` ways to represent them. That's easier on tooling and editors. あ is `u'\u3042'` and わ is `u'\u308f'`.

Granted this is really frustrating--it'd be much better if tools, editors, fonts, and such supported unicode better.

Reply:
I see! I will change it.